binary/test/test.hs

```diff
@@ -1 +1,20 @@
-{-# OPTIONS_GHC -F -pgmF hspec-discover #-}
+import           Universum
+
+import           Test.Hspec (hspec)
+
+import           Spec (spec)
+
+import qualified Test.Pos.Binary.BiSerialize
+
+main :: IO ()
+main = do
+    hspec spec
+    runTests
+        [ Test.Pos.Binary.BiSerialize.tests
+        ]
+
+runTests :: [IO Bool] -> IO ()
```

Comment:
This is an obvious duplication of https://github.com/input-output-hk/cardano-sl/blob/3feb70bfb399dfb8d1b2ba2730c044544e91738a/crypto/test/Test/Pos/Crypto/TempHelpers.hs#L60
Since that function is in `crypto`'s helpers, though, I wasn't sure if we should put it into a more general test utility package. Thoughts?

Reply:
Actually, the whole of that `TempHelpers.hs` module should be moved to the `binary` package (new name of `Helpers.hs` maybe) and then exposed via the `cardano-sl-binary-test` package where other package can import it.

Up to you whether you cant to make that part of this PR or submit another PR that then replaces `TempHelpers.hs` in `crypto` and other packages. In fact, I'm a little surprised you didn't need other functionality from the `TempHelpers` module.

Reply:
👍 I'll make the changes in a separate PR.